### PR TITLE
Change all the routes to v5

### PR DESCRIPTION
**Pull request by t16n**
I tried using staged-releases with openBalena but it didn't work with the v4 api url so i switched them to v5.

### CODE REVIEW

**Comment by thgreasi** [comment]
Hi @t16n and thanks for your contribution on this.
Everything seems fine to me but we will need you to change the commit description a bit so that our internal versioning system works and the `ResinCI/check-versionist/versionist` check passes.

We will need you to change (amend) your commit description to also contain `Change-type: major`.
Ideally the commit title should also start with a capital present verb.

In short, when you do a `git log` you should be getting something like:
```
Change all the routes to v5

Change-type: major
```

Give me a note if you have any issues while trying this or you prefer that we handle those changes on our side.

Once again, thanks for your contribution on this.
This is definitely going to be useful for other users.

Kind regards,
Thodoris

**Comment by t16n** [comment]
> We will need to add `Change-type: major` to the commit description, but otherwise this LGTM 👍

Ok, changed the commit message

**Comment by thgreasi** [comment]
Once again @t16n thanks for your contribution on this.
Should be useful for other open-balena users that want to have a staged releases schema by using sh scripts.
